core/execution/spot/market.go

```diff
@@ -492,6 +492,11 @@ func (m *Market) EnterLongBlockAuction(ctx context.Context, duration int64) {
 		return
 	}
 
+	// markets in governance auction are unaffected by long block auctions.
+	if m.mkt.TradingMode == types.MarketTradingModeSuspendedViaGovernance {
+		return
+	}
+
 	m.mkt.State = types.MarketStateSuspended
 	m.mkt.TradingMode = types.MarketTradingModelLongBlockAuction
```

Comment:
I think this is still wrong you know? or at least out of place. If we're not in an auction then it's correct. But if we are in an auction I'm not sure we should do either of these.